server/routes/admin.js

```diff
@@ -35,9 +35,30 @@ router.post('/categories/add', function (req, res, next) {
     res.redirect("/");
     res.end()
   }
-  categoryClient.addCategory(query, callback);
+  categoryClint.addCategory(query, callback);
 });
 
+/*===========================
+        Add article
+===========================*/
+router.get('/articles/add', (req, res, next) => {
```

Comment:
you have two routes for handling /articles/add now - the new one, and the one below (line 62), which one is correct? remove the other one